src/test-suite/spec.ts

```diff
@@ -203,5 +207,27 @@ export const runTestSuite = (params: TestSuiteParams) => {
 
 		});
 
+		describe('attr', () => {
```

Comment:
to be consistent with other tests
```suggestion
		describe('attr()', () => {
```